func1.js

```diff
@@ -0,0 +1,15 @@
+const runningAverage = () => {
+    let counter = 0;
+    let number = 0;
+
+    return (input) => {
+      number++;
+      counter+=input;
+      return counter/number;
+    };
+  };
+
+const rAvg = runningAverage();
+console.log(rAvg(10));
+console.log(rAvg(11));
+console.log(rAvg(12));
```

Comment:
Please, don't commit console.log and any other data which is used for testing purposes :)